client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -309,7 +309,7 @@ export default React.createClass( {
 			<SelectDropdown compact
 					className="plugin-list-header__actions_dropdown"
 					key="plugin-list-header__actions_dropdown"
-					selectedText={ this.translate( 'Actions' ) } >
```

Comment:
Hi! I've found a possible matching string that has already been translated **92** times:
`translate( 'Actions' )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/tests/localci-test-branch)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.